### PR TITLE
Example: Run Secretlint only on changed files with Node.js or Docker

### PR DESCRIPTION
**Pull request by gkzz**
The followings are example workflows of https://github.com/secretlint/secretlint/issues/166

### CODE REVIEW

File: .github/workflows/secretlint-only-on-changed-files.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        uses: actions/checkout@v3
+      - uses: technote-space/get-diff-action@v6
+      - name: setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: Install
+        run: npm install
+      - name: Lint with Secretlint
+        run: npx --no-install secretlint ${{ env.GIT_DIFF_FILTERED }}
```
Comment (gkzz) [inline .github/workflows/secretlint-only-on-changed-files.yml:22]: This workflow's output is as bellows;
https://github.com/gkzz/secretlint-github-actions-example/actions/runs/2043608456

File: .github/workflows/secretlint-via-docker-only-on-changed-files.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        uses: addnab/docker-run-action@v3
+        with:
+          # https://hub.docker.com/r/secretlint/secretlint/tags
+          image: secretlint/secretlint:v5.1.1
+          # https://github.com/secretlint/secretlint#using-docker
+          options: >
+            -v ${{ github.workspace }}:${{ github.workspace }}
+            -w=${{ github.workspace }}
+            --rm
+          run: secretlint ${{ env.GIT_DIFF_FILTERED }} # run only on changed files
```
Comment (gkzz) [inline .github/workflows/secretlint-via-docker-only-on-changed-files.yml:24]: This workflow's output is as bellows;
https://github.com/gkzz/secretlint-github-actions-example/runs/5701493132?check_suite_focus=true

File: .github/workflows/secretlint.yml [hunk 3]
```diff
@@ -12,7 +12,7 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v3
         with:
-          node_version: 16
+          node-version: 16
```
Comment (gkzz) [inline .github/workflows/secretlint.yml:15]: setup-node action.yml says
https://github.com/actions/setup-node/blob/bed538bd04a773f1616efba082915a7638d8eb5f/action.yml#L8

File: aws-secrets [hunk 4]
```diff
@@ -0,0 +1 @@
+AWS_SECRET_ACCESS_KEY = wJalrXUtnFEMI/K7MDENG/bPxRfiCYSECRETSKEY
```
Comment (gkzz) [inline aws-secrets:1]: Copied from the same file on `aws-secret` branch
https://github.com/secretlint/secretlint-github-actions-example/blob/1729c7d6f3340734a1936b4d98192c04fba8db31/aws-secrets

**Comment by gkzz** [comment]
@azu 
Could you please reoepn this pr?
The reason I closed before was the following `git-diff` related error happend to me, so that I need to delete and create a branch.
https://github.com/gkzz/secretlint-github-actions-example/actions/runs/2043594245

This cause of error is I did `git rebase` on my local

Thanks, always
gkzz

**Comment by gkzz** [comment]
> @azu Could you please reoepn this pr? The reason I closed before was the following `git-diff` related error happend to me, so that I need to delete and create a branch. https://github.com/gkzz/secretlint-github-actions-example/actions/runs/2043594245
> 
> This cause of error is I did `git rebase` on my local
> 
> Thanks, always gkzz

According to the similar issue, [command [git diff] exited with code 128. message: fatal: Invalid symmetric difference expression #178](https://github.com/technote-space/get-diff-action/issues/178#issuecomment-1022655067) ,
if we make a use of [technote-space/get-diff-action: GitHub Actions to get git diff](https://github.com/technote-space/get-diff-action) , it may be better to add `refrain from using forced push` to the document.

**Comment by azu** [comment]
> Could you please reoepn this pr?

image

It looks like I can not it.

**Comment by gkzz** [comment]
> > Could you please reoepn this pr?
> 
> image
> 
> It looks like I can not it.

Same here!
Thanks for your quick response!!
I'll try make another pr with a new branch, whose name is `secretlint-only-on-changed-files-with-nodejs-and-docker`

**Pull request closed without merging**